### PR TITLE
added alert component

### PR DESCRIPTION
**Pull request by Hamza-777**
Please provide feedback if any and do review this PR. Thank you.

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.alert-danger {
+    background: #FF6464;
+    color: #fff;
+    border: none;
+}
+
+.alert-warning {
+    background: #FFE162;
```
Comment (partha8) [inline components/alert.css:20]: You should use variables for colors as it is a good practice.

File: docs.html [hunk 2]
```diff
@@ -49,94 +49,215 @@ <h1>AVATAR</h1>
...
+                        </div>
+                        <div class="line">
+                            <span class="comment">/* Success Alert */</span>
+                        </div>
+                        <div class="line">
+                            <span class="sym"><</span>
+                            <span class="element">div </span>
+                            <span class="attribute">class</span>
+                            <span class="sym">=</span>
+                            <span class="attr-content">"alert alert-success"</span>
```
Comment (partha8) [inline docs.html:222]: Use descriptive class names as it improves readability and understanding.

**Comment by hsnice16** [comment]
Nice work, Hamza. Just do the changes that @partha8 suggested.

File: docs.html (hunk 2 above)
Comment (Hamza-777) [reply docs.html:222]: Okay thanks partha

File: components/alert.css (hunk 1 above)
Comment (Hamza-777) [reply components/alert.css:20]: sure thing thanks

**Pull request merged**